conf/application.conf

```diff
@@ -130,7 +130,9 @@ self-assessment-enrolments = "enr3,enr4"
 company-auth.host = "http://localhost:9025"
 contact-frontend.host = "http://localhost:9250"
 tax-account-router.host = "http://localhost:9280"
-business-enrolments = "enr1,enr2"
+business-enrolments = "enr1, enr2, IR-SA"
```

Comment:
Imho, better to put a definite dummy value in to intersect here, rather than an actual valid enrolment identifier